components/eventing-controller/controllers/subscription/nats/reconciler_test.go

```diff
@@ -117,160 +117,79 @@ func testCreateDeleteSubscription(id int, eventTypePrefix, natsSubjectToPublish,
 }
 
 func testCreateSubscriptionWithInvalidSink(id int, eventTypePrefix, natsSubjectToPublish, eventTypeToSubscribe string) bool {
```

Comment:
You are no longer checking this in your test:
```
Expect(backendSubscription.Subject).Should(Equal(natsSubjectToPublish))
```
if this is ententional you could change the functions signature  to
```suggestion
func testCreateSubscriptionWithInvalidSink(id int, eventTypePrefix, _, eventTypeToSubscribe string) bool {
```
and still fulfill the `testCase` requirements